src/schema/v2/artwork/index.ts

```diff
@@ -1665,19 +1672,26 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
           sort: { type: ShowSorts },
         },
         resolve: (
-          { id },
+          { id, show_ids },
           { active, sort, atAFair: at_a_fair },
-          { relatedShowsLoader }
+          { showsLoader }
         ) =>
-          relatedShowsLoader({
-            artwork: [id],
-            size: 1,
-            active,
-            sort,
-            at_a_fair,
-          })
-            .then(({ body }) => body)
-            .then(_.first),
+          show_ids && show_ids.length > 0
+            ? showsLoader(
+                _.omitBy(
```

Comment:
I don't get the `omitBy` and `isNil` thing - i believe `status: null` would get left off the upstream request by default (prob don't need to manually do it).

Reply:
It didn't work for me. The empty `status` parameter is sent to the API, which rejects it because it's not in the allowed values. Is there a more concise of stripping `null` properties here?

Reply:
`compact` maybe

Reply:
There's some helpers too - `removeNulls` for one - https://github.com/artsy/metaphysics/blob/70fd9642465eaaaceff2babb493d0f60d7a7fd84/src/lib/helpers.ts#L184